packages/react-app/src/components/common/BridgeDropdown.jsx

```diff
@@ -0,0 +1,34 @@
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import { Button, Menu, MenuButton, MenuItem, MenuList } from '@chakra-ui/react';
+import { Web3Context } from 'contexts/Web3Context';
+import { bridgeNames } from 'lib/constants';
+import { getBridgeName, getBridgeNetwork, getBridgeUrl } from 'lib/helpers';
+import React, { useContext, useState } from 'react';
+
+export const BridgeDropdown = () => {
+  const { providerChainId: chainId } = useContext(Web3Context);
+  const bridgeChainId = getBridgeNetwork(chainId);
+  const defaultBridgeUrl = getBridgeUrl(`${chainId}-${bridgeChainId}`);
+  const defaultBridgeName = getBridgeName(defaultBridgeUrl);
+  const [bridgeName, setBridgeName] = useState(defaultBridgeName);
+
+  const changeItem = e => {
+    setBridgeName(e.target.name);
+    window.location.href = e.target.dataset.url;
```

Comment:
Is the end goal of this issue to redirect here? One question I have is how do we want to handle the different urls when developing locally?